### ISSUE TITLE
New project generates errors

### ISSUE THREAD

**Issue by ariejan** [kabisa/maji#229]
## How to reproduce

 * Create a new Maji app: `yarn create maji-app com.example.myapp ~/Code/myapp`
 * Start Maji: `bin/maji start` and open http://localhost:9090
 * Check browser console

```
GET http://localhost:9090/cordova.js net::ERR_ABORTED
GET http://localhost:9090/__webpack_hmr 404 (Not Found)
```

<img width="736" alt="splinter" src="https://user-images.githubusercontent.com/1913/36969688-8bf435f2-2066-11e8-991a-7e3063df5785.png">

**Comment by pascalw** [comment]
I think @matthijsgroen just fixed exactly this, #227, #228. Can you retry with master?

**Comment by pascalw** [comment]
@ariejan I've just published `3.3.0`. Can you confirm this is now fixed?

**Comment by pascalw** [comment]
Closing due to inactivity. @ariejan if you still encounter issues feel free to re-open.
